app/controllers/maintainers_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class MaintainersController < ApplicationController
+  before_action :load_project
+  def edit
+  end
+
+  def create
+    user = User.find_by(id: params[:maintainer_id])
+    @project.maintainers << user
+    redirect_to edit_project_maintainers_path
+  end
+
+  def destroy
+    user = User.find_by(id: params[:maintainer_id])
+    @project.maintainers.destroy(user)
```

Comment:
A komment az előző akciónál itt is érvényes: legyen lekezelve, ha hibára futunk.